iconservice/icon_service_engine.py

```diff
@@ -637,7 +637,7 @@ def _update_productivity(cls,
             Logger.warning(tag=cls.TAG, msg=f"No block validators: block={context.block}")
             return
 
-        validators: List[Tuple['Address', bool]] = [[prev_block_generator, True]]
+        validators: List[Tuple['Address', int]] = [[prev_block_generator, True]]
```

Comment:
Typehint and value mismatch
True is not int format

Reply:
I fix what I missed about that.